doc/rados/api/python.rst

```diff
@@ -319,6 +319,13 @@ exists,  and delete a pool.
 .. automethod:: Rados.delete_pool(pool_name)
 
 
+Mon Commands
```

Comment:
Would "CLI Commands" be a better heading? The term "Mon Commands" seems potentially confusing.

Reply:
fixed

Reply:
i am not sure CLI commands is better. ceph cli is implemented using these bindings. it's like documenting some GTK+ APIs for drawing widgets with the title of 

> GNOME Applications